cpp/src/io/avro/avro_gpu.cu

```diff
@@ -75,11 +75,15 @@ avro_decode_row(schemadesc_s const* schema,
                 uint8_t const* end,
                 device_span<string_index_pair const> global_dictionary)
 {
+  using namespace cudf::io::avro;
```

Comment:
is this needed? AFAICT, this code is in cudf::io::avro::gpu so anything in cudf::io::avro should be visible.

Reply:
Fixed, pushed.